docs/hints-for-use-with-docker.md

```diff
@@ -12,7 +12,18 @@ docker run \
   analyze [...] # Insert further arguments for the command.
 ```
 
-**Note:** The single forward slash `/` between the environment variable `$PWD` and the `:` is required for PowerShell compatibility, as PowerShell otherwise interprets `:` as part of the environment variable. 
+Make sure to mount the project root directory so ORT is able to detect VCS information. To work on a subproject use the option `-i` to specify the actual working directory:
+
+```shell
+docker run \
+  -v $PWD/:/project \ # Mount current working directory into /project to use as input.
+  ort --info \
+  -c /project/ort/config.hocon \ # Use file from "<workingdirectory>/ort" as config.
+  -i /project/subproject \ # Specify working directory
+  analyze [...] # Insert further arguments for the command.
+```
```

Comment:
In general, I wonder whether there's a nice way to combine this example with the previous one. For example, if we extended the first example's line 12 by writing

    analyze -i /project [...] # Insert further arguments for the command.

we could simply add this sentence instead of a second mostly identical example:

> If only a subproject shall be analyzed, change `-i /project` to `-i /project/subproject`. Note that still the project root directory needs to be mounted to Docker for ORT to detect VCS information.

Or something like that.

Reply:
Good idea, changed the docuumentation accordingly.

Reply:
Thanks! One (or actually three) last wish(es): Please squash your commits (we prefer commits to be amended to address review comment, not new commits to be added to a PR), rebase onto `origin/main` (note that we've changed the default branch from `master` to `main` recently), and force-push your branch.